resources/windows/msi/service.mcollective.wxs

```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <DirectoryRef Id="TBD">  # The ID has to match the Ruby Bin directory
+      <Component Id='MCOService' Guid="7601FCEA-90B3-CC69-6A69-4087FBC7292D" Win64="$(var.Win64)">
+        <File Id="RubyWExe" KeyPath="yes" Source="stagedir\sys\ruby\bin\rubyw.exe" />
+        <!-- This service is installed with start set to demand because
+        it won't be correctly configured to start right away. The
+        puppet run that configures mcollective will allow it to start
+        and set it to automatic -->
+        <ServiceInstall Id="MCOServiceInstaller"
+          Account="[PUPPET_AGENT_ACCOUNT_DOMAIN]\[PUPPET_AGENT_ACCOUNT_USER]"
+          Password="[PUPPET_AGENT_ACCOUNT_PASSWORD]"
+          Description="Puppet Labs server orchestration framework"
+          DisplayName="Marionette Collective Server"
+          Interactive="no"
+          Name="mcollective"
+          Start="demand"
+          Type="ownProcess"
+          ErrorControl="normal"
+          Vital="yes"
+          Arguments="-I&quot;[INSTALLDIR]puppet\lib;[INSTALLDIR]facter\lib;[INSTALLDIR]hiera\lib;[INSTALLDIR]mcollective\lib&quot; -rubygems &quot;[INSTALLDIR]mcollective\bin\mcollectived&quot; --daemonize"
```

Comment:
If https://github.com/puppetlabs/puppet-agent/pull/542 gets merged, this should be `"-I&quot;[INSTALLDIR]puppet\lib;&quot; -rubygems &quot;[INSTALLDIR]puppet\bin\mcollectived&quot; --daemonize"`